# ALL:ALL - Se reemplaza la referencia directa a pthread por Threads::Threads y se elimina -lpthread en pc_file para que funcione el enlace de hilos portable en CMake.
--- libuavs3d/source/CMakeLists.txt
+++ libuavs3d/source/CMakeLists.txt
@@ -136,7 +136,7 @@
 file(APPEND "${pc_file}" "Description: AVS3 decoder library \n")
 file(APPEND "${pc_file}" "Version: ${uavs3d_version}\n")
 if(CMAKE_USE_PTHREADS_INIT)
-  file(APPEND "${pc_file}" "Libs: -L\${libdir} -l${LIBNAME} -lm -lpthread\n")
+  file(APPEND "${pc_file}" "Libs: -L\${libdir} -l${LIBNAME} -lm\n")
 else()
   file(APPEND "${pc_file}" "Libs: -L\${libdir} -l${LIBNAME} -lm\n")
 file(APPEND "${pc_file}" "Libs.private: \n")
@@ -155,7 +155,7 @@
     target_link_libraries(${LIBNAME} m)
 endif()
 if(CMAKE_USE_PTHREADS_INIT)
-target_link_libraries(${LIBNAME} pthread)
+target_link_libraries(${LIBNAME} Threads::Threads)
 endif()
 
 # install
# ALL:ALL - Se reemplaza logica original para adaptarla al sistema con parches estaticos para que funcione la compilacion sin sed y cruce correctamente.
--- /dev/null
+++ libuavs3d/version.h
@@ -0,0 +1,9 @@
+#ifndef __VERSION_H__
+#define __VERSION_H__
+#define VER_MAJOR  1
+#define VER_MINOR  0
+#define VER_BUILD  0
+#define VERSION_TYPE "release"
+#define VERSION_STR  "1.0.0"
+#define VERSION_SHA1 "unknown"
+#endif // __VERSION_H__
